src/schema.js

```diff
@@ -268,7 +268,9 @@ const updateUserSchema = Joi.object({
 
 const fetchPostsSchema = Joi.object({
 	query: {
-		isFlagged :Joi.boolean()
+		isFlagged :Joi.boolean(),
+		cursor : Joi.number(),
+		limit : Joi.number().default(20)
```

Comment:
we can let this be without the default in light of the other suggested change

Reply:
sure, thanks